libmamba/src/core/query.cpp

```diff
@@ -800,12 +881,13 @@ namespace mamba
                 auto package = m_dep_graph.node(id);
                 packages[package.name].push_back(package);
             }
-
             for (const auto& entry : packages)
             {
                 print_solvable(
+                    out,
                     entry.second[0],
-                    std::vector(entry.second.begin() + 1, entry.second.end())
+                    std::vector(entry.second.begin() + 1, entry.second.end()),
+                    Context::instance().output_params.verbosity > 0
```

Comment:
This is a slight difference to the original patch #2823; since the `context` singleton isn't available in the entrypoint in libmamapy yet, we're retrieving the context singleton here directly.